cl/opinion_page/templates/publish.html

```diff
@@ -37,7 +37,11 @@
     <div class="row">
       <div class="col-xs-1"></div>
       <div class="col-xs-2">
-        <img src="{% static "img/tn-court.jpg" %}" class="img-responsive" width="359" height="359" >
+        {% if form.initial.court.id == "me" %}
+          <img src="https://seals.free.law/v2/128/me.png" class="img-responsive" height=128 width="128" />
+        {% else %}
```

Comment:
Being explicit in this else statement seems good too.

Reply:
Great.